### PR TITLE
Install Google Cloud SDK in the driver image

### PR DESCRIPTION
**Pull request by lgruen**
Fixes the currently broken build: https://github.com/populationgenomics/analysis-runner/runs/5460052892?check_suite_focus=true

### CODE REVIEW

File: driver/Dockerfile [hunk 1]
```diff
@@ -6,28 +6,30 @@ ENV MAMBA_ROOT_PREFIX /root/micromamba
...
-        phantomjs \
-        r-argparser \
-        r-base=4.1.1 \
-        r-essentials \
-        r-tidyverse \
-        selenium \
-        skopeo \
-        statsmodels && \
+    -c cpg -c bioconda -c conda-forge \
+    bokeh \
```
Comment (illusional) [inline driver/Dockerfile:19]: Is there a reason for removing this formatting? I think it's cleaner as it shows it's arguments for the micromamba install.

**Review by illusional — APPROVED** [review]
The change looks fine to me, do we know why it started spontaneously failing?
I left a comment about removing the indentation, my personal preference is to keep the indentation.

**Comment by lgruen** [comment]
> The change looks fine to me, do we know why it started spontaneously failing? I left a comment about removing the indentation, my personal preference is to keep the indentation.

I don't think it's spontaneous -- this has been broken for a while, probably in the context of the recent conda --> pip changes: https://github.com/populationgenomics/analysis-runner/actions/workflows/hail_update.yaml
CleanShot 2022-03-08 at 17 14 22@2x

File: driver/Dockerfile (hunk 1 above)
Comment (lgruen) [reply driver/Dockerfile:19]: That's just what VS Code does auto-formats Dockerfiles for me... Don't have a strong opinion myself.

Comment (lgruen) [reply driver/Dockerfile:19]: Let me try https://github.com/microsoft/vscode-docker/pull/2976

**Pull request merged**